gmso/external/convert_hoomd.py

```diff
@@ -0,0 +1,1439 @@
+"""Convert GMSO Topology to GSD snapshot."""
+from __future__ import division
+
+import itertools
+import json
+import re
+import statistics
+import warnings
+
+import numpy as np
+import unyt as u
+from unyt.array import allclose_units
+
+from gmso.core.views import PotentialFilters
+from gmso.exceptions import NotYetImplementedWarning
+from gmso.lib.potential_templates import PotentialTemplateLibrary
+from gmso.utils.conversions import (
+    convert_opls_to_ryckaert,
+    convert_ryckaert_to_opls,
+)
+from gmso.utils.geometry import coord_shift
+from gmso.utils.io import has_gsd, has_hoomd
+from gmso.utils.sorting import (
+    natural_sort,
+    sort_connection_members,
+    sort_member_types,
+)
+
+if has_gsd:
+    import gsd
```

Comment:
I think this should be `import gsd.hoomd`  I'm getting an attribute error if we only import gsd then try calling `gsd.hoomd`.